server/services/cryptoDonationService.js

```
@@ -0,0 +1,51 @@
+import { ethers } from 'ethers';
```

ESLint

[error] Replace `'ethers'` with `"ethers"`

(prettier/prettier)